input_output.ipynb

```diff
@@ -131,12 +197,27 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
```

Comment:
I think here there is a missing title and should be "Quiz on string input"

Reply:
Done in ca7e073